### PR TITLE
Create webkitcssmatrix.json

### PR DESCRIPTION
**Pull request by Yaffle**
https://github.com/Fyrd/caniuse/issues/1001

### CODE REVIEW

File: features-json/webkitcssmatrix.json [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    "DOM"
+  ],
+  "stats":{
+    "ie":{
+      "5.5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"x #1",
```
Comment (Fyrd) [inline features-json/webkitcssmatrix.json]: These need an included `y` or `a` value

**Comment by Fyrd** [comment]
Thanks! Few notes:
- The `x` values need a `y` or `a` included to specify support or partial support.
- Can you rename the file to "dommatrix.json" since DOMMatrix is the standard?

**Comment by Yaffle** [comment]
@Fyrd, the situation is similar to "-webkit-text-stroke", when the prefix "webkit" is a de facto standard....
but OK, I will change the filename.

**Comment by Fyrd** [comment]
Thanks! I'll merge this as it is right now, though I think we'll want additional notes to clarify which prefixed variation is supported. But I'll be happy to help out with that too if you like.

**Pull request merged**